kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/TestUtils.java

```diff
@@ -88,7 +88,7 @@ public static boolean isValidJson(String value) {
      * @return the random pod name
      */
     public static String getRandomPodNameSuffix() {
-        return MobyNamesGenerator.getRandomName().replace("_", "-");
+        return MobyNamesGenerator.getRandomName(1).replace("_", "-");
```

Comment:
so the `retry` int doesn't control retries? It appends a random int between 0 and 10 inclusive? :exploding_head:

Reply:
yup...